### PR TITLE
Ensure that VB parenthesis simplification works for XML elements and XML empty elements

### PR DESCRIPTION
**Pull request by DustinCampbell**
Fixes Issue #4490

Reviewers: @balajikris, @brettfo, @CyrusNajmabadi, @davkean, @dpoeschl, @jasonmalinowski, @jmarolf, @Pilchie, @rchande

### CODE REVIEW

**Comment by balajikris** [comment]
LGTM.

**Comment by DustinCampbell** [comment]
Only the linux build failed with a Nuget package error that is unrelated to this change. Merging...

File: src/EditorFeatures/Test2/Simplification/ParenthesisSimplificationTests.vb [hunk 1]
```diff
@@ -1080,5 +1080,261 @@ End Module
...
+            Dim input =
+<Workspace>
+    <Project Language="Visual Basic" CommonReferences="true">
+        <Document>
+Imports System.Xml.Linq
+
+Class C
+    Sub M()
+        Dim x = 1 &gt; {|Simplify:(&lt;xml&gt;&lt;/xml&gt;)|}
+    End Sub
```
Comment (Pilchie) [inline src/EditorFeatures/Test2/Simplification/ParenthesisSimplificationTests.vb]: These xml literal tests inside xml literals make my eyes bleed. :smile:

**Comment by Pilchie** [comment]
:+1:

File: src/EditorFeatures/Test2/Simplification/ParenthesisSimplificationTests.vb (hunk 1 above)
Comment (DustinCampbell) [inline src/EditorFeatures/Test2/Simplification/ParenthesisSimplificationTests.vb]: That's your lasic surgery

**Pull request merged**